backend/auth/firebase.go

```diff
@@ -15,36 +15,56 @@
 package auth
 
 import (
-	"context"
-	"firebase.google.com/go/v4/auth"
-	"log"
-	"net/http"
-	"strings"
-	"time"
+  "context"
+  "firebase.google.com/go/v4/auth"
+  "net/http"
+  "strings"
+  "time"
 )
 
 type FirebaseAuth struct {
 	FirebaseClient firebaseAuthClient
 	AuthTimeout    time.Duration
+	AuthDomain     *string
```

Comment:
Why `*string` rather than just `string`?

Reply:
I struggled to get the tests to work because just doing `auth.AuthDomain = ""` doesn't seem to change the variable in the server (and/or handler?) so I thought using a pointer would make sense to allow the auth domain to be changed during runtime without the server having to be reinitialized. Seeing that that's neither something I have actually implemented nor something a user would likely want to do that was probably not a very useful consideration. I've changed it now.